### PR TITLE
Doc/download feature

### PR DESCRIPTION
**Pull request by dns2utf8-tipi**
Merges the existing docs with the new section for the `download` feature

Live: https://doc-download-feature.dev.tipi.build/documentation/10-tipi-cache

### CODE REVIEW

File: 10-tipi-cache.md [hunk 1]
```diff
@@ -33,6 +33,41 @@ Whenever you launch a build tipi will produce a mirror of your sources either lo
...
+```
+
+We can now build the sources remotely like this:
+```sh
+tipi -t linux-cxx17 build .
+```
+
+Finally we can download the complete binaries like this:
+```sh
+tipi -t linux-cxx17 download "./build/linux-cxx17/bin/src/string*"
```
Comment (daminetreg) [inline 10-tipi-cache.md:64]: I would also document just downloading a single file at first

**Review by daminetreg — CHANGES_REQUESTED** [review]

File: 10-tipi-cache.md (hunk 1 above)
Comment (dns2utf8-tipi) [reply 10-tipi-cache.md:64]: Should we document the variant without the `build/plattform/bin/` as well?

Comment (daminetreg) [reply 10-tipi-cache.md:64]: No it's fine this way thanks !

**Review by daminetreg — APPROVED** [review]

File: 10-tipi-cache.md [hunk 2]
```diff
@@ -33,6 +33,50 @@ Whenever you launch a build tipi will produce a mirror of your sources either lo
 
 If you want to fully ignore data for the build you can add them to the `.tipiignore` file.
 
+## Downloading remote build results
+
+With tipi `v0.0.51` you can download build results directly onto your machine without downloading the full (potentially very big) build directory.
```
Comment (pysco68) [inline 10-tipi-cache.md]: my only nitpick: the "(potentially very big)" is only a filler IMO plus it does sound very non-native (as in, we should at least use another adjective as the universally used "big" / "very big" qualifier).

I would remove it.

**Review by pysco68 — CHANGES_REQUESTED** [review]
Thanks for the great addition, I would only remove the "very big" thing and then we're good to 🚢 🚀

**Comment by Lambourl** [comment]
@dns2utf8-tipi Can you make the changes requested by @pysco68 ?
This will allow us to merge this pr

**Comment by dns2utf8-tipi** [comment]
Thanks for the review. All done :rainbow:

**Pull request merged**